### PR TITLE
Mapper Tests + CourierInfoDtoMapper fix

### PR DESCRIPTION
**Pull request by SomeOne35719**
Mapper Tests + CourierInfoDtoMapper fix

### CODE REVIEW

**Review by Natalia1910 — APPROVED** [review]

**Review by Max877866 — APPROVED** [review]

**Review by Pavl0-Gural — APPROVED** [review]

**Review by Andrii-Yezenitskyi — APPROVED** [review]

File: service/src/test/java/greencity/mapping/EmployeeToOptionForColumnDtoMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.springframework.transaction.annotation.Transactional;
+
+@ExtendWith(MockitoExtension.class)
+class EmployeeToOptionForColumnDtoMapperTest {
+
+    @InjectMocks
+    EmployeeToOptionForColumnDtoMapper employeeToOptionForColumnDtoMapper;
+
+    @Test
+    @Transactional
```
Comment (thesvider) [inline service/src/test/java/greencity/mapping/EmployeeToOptionForColumnDtoMapperTest.java:20]: what's the purpose of Transactional annotation in mock test?

File: service/src/test/java/greencity/mapping/OrderAddressDtoUpdateMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            .street("Gorodotska")
+            .houseCorpus("2")
+            .entranceNumber("7a")
+            .houseNumber("7")
+            .build();
+
+        OrderAddressDtoResponse actual = orderAddressDtoUpdateMapper.convert(address);
+
+        Assertions.assertEquals(actual.toString(), expected.toString());
+
```
Comment (thesvider) [inline service/src/test/java/greencity/mapping/OrderAddressDtoUpdateMapperTest.java:34]: not good idea to check toString value. check specific fields instead.

File: service/src/test/java/greencity/mapping/EmployeeToOptionForColumnDtoMapperTest.java (hunk 1 above)
Comment (SomeOne35719) [reply service/src/test/java/greencity/mapping/EmployeeToOptionForColumnDtoMapperTest.java:20]: I will fix it in the next commit

File: service/src/test/java/greencity/mapping/OrderAddressDtoUpdateMapperTest.java (hunk 2 above)
Comment (SomeOne35719) [reply service/src/test/java/greencity/mapping/OrderAddressDtoUpdateMapperTest.java:34]: I will change the way of ckecking

**Pull request merged**